### ISSUE TITLE
The Next and Back button on the Help tour don't have have enough contrast

### ISSUE THREAD

**Issue by joshuadkitenge** [ral-facilities/scigateway#790]
**Description**:
## Light Mode
The Next and Back button text doesn't have enough contrast to satisfy WCAG 2 AA.  Suggested foreground colour change #c34f00 from #ff6900

![help_tour_buttons(Light_mode)](https://user-images.githubusercontent.com/83226114/139655187-45e989c3-f955-4180-baf2-3f8c235bd997.png)

<br><br>

## Dark Mode

The Next button text doesn't have enough contrast to satisfy WCAG 2 AA.  Suggested foreground colour change #c34f00 from #ff6900

![help_tour_buttons(dark_mode)](https://user-images.githubusercontent.com/83226114/139655431-231390a7-c8fd-4ca2-8b85-66cc0653e811.png)

**Acceptance criteria**:
- [ ] A checklist
- [ ] of criteria to accept this story as done
